StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/BottomSheetViewController.swift

```diff
@@ -221,6 +221,9 @@ class BottomSheetViewController: UIViewController, BottomSheetPresentable {
         contentContainerView.addArrangedSubview(self.contentViewController.view)
         if let presentationController = rootParent.presentationController as? BottomSheetPresentationController {
             presentationController.forceFullHeight = newContentViewController.requiresFullScreen
+
+            // Force layout pass to apply safe area insets
+            presentationController.containerView?.layoutIfNeeded()
```

Comment:
This works, but I'm sure there's a better solution than forcing a layout pass. Let me know if you have any ideas!